src/odemis/acq/align/transform.py

```diff
@@ -94,3 +99,315 @@ def CalculateTransform(optical_coordinates, electron_coordinates, skew=False):
         shear_ret = -shear
 
         return (translation_xy_ret[0], translation_xy_ret[1]), (scaling_ret, scaling_ret), theta_ret, scaling_xy_ret, shear_ret
+
+
+class Transform(object):
```

Comment:
Let's move this class to odemis.util. I'm not sure which module is best. Maybe `conversion.py`, or a new module.

Reply:
`conversion.py` doesn't really seem to fit, I'll put it in a new module.